package-lock.json

```diff
@@ -6547,6 +6547,7 @@
             "version": "4.0.5",
             "resolved": "https://registry.npmjs.org/bufferutil/-/bufferutil-4.0.5.tgz",
             "integrity": "sha512-HTm14iMQKK2FjFLRTM5lAVcyaUzOnqbPtesFIvREgXpJHdQm8bWS+GkQgIkfaBYRHuCnea7w8UVNfwiAQhlr9A==",
+            "hasInstallScript": true,
```

Comment:
Why is this change in this PR?

Reply:
Not sure how that was added; it must have been when I did run `npm ci` or something. Will revert.